setup.py

```diff
@@ -32,7 +32,11 @@
     license='All Rights Reserved.',
     packages=find_packages(),
     # dependencies are named in requirements.pip
-    install_requires=[],
+    install_requires=[
```

Comment:
They are also listed in `requirements.pip`. Listing them out here will allow people to just install this package and get the requirements. It is not as good as `requirements.pip` as it will just grab whatever the latest version is, but I think it is a good practice to do so.

cc: @plathrop-sfdc